### DISCUSSION TITLE
Mass-unmuting tests

### DISCUSSION THREAD

**Discussion by Neboyarsky** [allure-framework/allure2#2403] [TestOps]
The other day I mistakenly muted 55 tests and spent about 10 minutes to unmute them all one by one. Having a way to mass-unmute all tests that are muted for the same reason, would help tremendously! 
I have searched for similar discussions [to no luck](https://github.com/orgs/allure-framework/discussions?discussions_q=Mass-unmuting+tests), surpprisingly!

**Comment by cheshi-mantu** [comment]
Hi @Neboyarsky ,
if you feel some of the UX in Allure Testops could be improved, please do not hesitate to file a Feature request at https://help.qameta.io.

cheers

**Reply by Neboyarsky** [reply]
Hi @cheshi-mantu, I honestly tried but could not find a place at help.qameta.io to submit feedback - all I found is a collection of help pages. Do you mind helping to find the place where to do it?

**Reply by cheshi-mantu** [reply]
you need to register at the help desk first
![image](https://github.com/allure-framework/allure2/assets/8607348/4d023860-3df8-4037-a89f-d24dd39fc2e7)
